settings.py

```diff
@@ -42,6 +42,7 @@
     'control',
     'control.users',
     'import_export',
+    'control.purchase_order'
```

Comment:
Put this one right next to the others(`control` and `control.users`)

You can move the `import_export` and put it with the external libraries